### PR TITLE
Feature/#195 update path colliders

### PR DESCRIPTION
**Pull request by leroykorterink**
This should be merged after wave count PR.

### CODE REVIEW

File: src/Assets/LightSkyDefense/Scripts/UI/Scoreboard.cs [hunk 1]
```diff
@@ -5,19 +5,31 @@
 public class Scoreboard : MonoBehaviour
 {
     private AudioSource _audioSource;
```
Comment (Maxboef) [inline src/Assets/LightSkyDefense/Scripts/UI/Scoreboard.cs]: Private above public

File: src/Assets/LightSkyDefense/Scripts/Controls/PauseMenu/PauseMenuBehaviour.cs [hunk 2]
```diff
@@ -26,7 +20,10 @@ void Update()
             return;
         }
 
-        if(_menuInstance != null)
+        Debug.Log("Open");
```
Comment (mserier) [inline src/Assets/LightSkyDefense/Scripts/Controls/PauseMenu/PauseMenuBehaviour.cs]: Random log?

File: src/Assets/LightSkyDefense/Scripts/Path/Path.cs [hunk 3]
```diff
@@ -12,6 +12,9 @@ public class Path : MonoBehaviour
     [Tooltip("Amount of points to generate that form the line")]
     public int WaypointCount = 100;
 
+    [Tooltip("The radius that the capsule colliders in the path will get")]
+    private float PathWidth = 0.15f;
```
Comment (mserier) [inline src/Assets/LightSkyDefense/Scripts/Path/Path.cs]: _pathWidth

Comment (Maxboef) [inline src/Assets/LightSkyDefense/Scripts/Path/Path.cs]: Private above Public?

**Review by mserier — CHANGES_REQUESTED** [review]
I haven't tested the branch yet.

File: src/Assets/LightSkyDefense/Scripts/Path/Path.cs (hunk 3 above)
Comment (leroykorterink) [reply src/Assets/LightSkyDefense/Scripts/Path/Path.cs]: I'm not going to change the name since this field should be public.

**Review by mserier — APPROVED** [review]

**Pull request merged**